.github/workflows/merge-checks.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   merge-checks:
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
i would do the enviroments/matrix thing instead of this direct replacement

Reply:
Oh yeah, I was just testing if it works.
Probably worth adding a matrix to ensure it stays working though.

Reply:
Added a matrix.